tacacs_plus/__init__.py

```diff
@@ -220,10 +242,10 @@ def __str__(self):
 
 class TACACSAuthenticationStart(object):
 
-    def __init__(self, username, authen_type, data=six.b('')):
+    def __init__(self, username, authen_type, priv_lvl=TAC_PLUS_PRIV_LVL_MIN, data=six.b('')):
```

Comment:
Although, this parameter is related to authorization, it is required as a part of authentication start packet. I assumed that some server implementation might address it even on authentication. So, my suggestion is to allow passing it as a parameter.